app/models/card.rb

```diff
@@ -10,6 +10,17 @@ class Card < ActiveRecord::Base
   before_validation :define_review_date
   validates :original_text, :translated_text, presence: true
   validates_with CompareValidator
+  scope :time_to_check_card, -> { where("review_date < ?", Time.now).order("RANDOM()")}
```

Comment:
Line is too long. [87/80]<br>Space missing inside }.<br>Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.